InternetHospital.WebApi/appsettings.json

```diff
@@ -1,6 +1,8 @@
 {
   "ConnectionStrings": {
-    "DefaultConnection": "User ID = postgres;Password=12345678;Server=localhost;Port=5432;Database=HospitalDB;Integrated Security=true; Pooling=true;"
+    "LocalPostgreSqlConnection": "User ID = postgres;Password=12345678;Server=localhost;Port=5432;Database=HospitalDB;Integrated Security=true; Pooling=true;",
```

Comment:
we already moved to azure ms sql. do we need LocalPostgreSqlConnection?

Reply:
Done.